lib/core/src/Cardano/Pool/DB/Sqlite.hs

```diff
@@ -837,6 +856,21 @@ fromPoolProduction (PoolProduction pool slot headerH parentH height) =
         }
     )
 
+mkByronHeader
+    :: BlockHeader
+    -> ByronHeaders
+mkByronHeader block = ByronHeaders
+    { byronSlot = view #slotNo block
+    , byronHeaderHash = BlockId (headerHash block)
+    , byronParentHash = BlockId (parentHeaderHash block)
+    , byronBlockHeight = getQuantity (blockHeight block)
+    }
+
+fromByronHeaders :: ByronHeaders -> BlockHeader
+fromByronHeaders ByronHeaders { .. } =
+    BlockHeader byronSlot (Quantity byronBlockHeight) (getBlockId byronHeaderHash)
+        (getBlockId byronParentHash)
```

Comment:
https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters

```suggestion
    BlockHeader byronSlot
        (Quantity byronBlockHeight)
        (getBlockId byronHeaderHash)
        (getBlockId byronParentHash)
```